UPGRADE-1.9.md

```diff
@@ -47,23 +55,68 @@
 1. `/new-api` prefix has been changed to `/api/v2`. Please adjust your routes accordingly.
    Admin API is hardcoded to `/api/v1` instead of `/api/v{version}`.
 
-### New API
-
-1. Add new parameters, new access control configuration and reorder it:
+1. Add proper redirect to changing password page in your `config/routes/sylius_shop.yaml` file:
 
     ```diff
-        parameters:
-    +       sylius.security.new_api_user_account_route: "%sylius.security.new_api_shop_route%/account"
-    +       sylius.security.new_api_user_account_regex: "^%sylius.security.new_api_user_account_route%"
-
-        security:
-            access_control:
-    +           - { path: "%sylius.security.new_api_user_account_regex%/.*", role: ROLE_USER }
-    -           - { path: "%sylius.security.new_api_shop_regex%/.*", role: IS_AUTHENTICATED_ANONYMOUSLY }
-                - { path: "%sylius.security.new_api_route%/shop/authentication-token", role: IS_AUTHENTICATED_ANONYMOUSLY }
-    +           - { path: "%sylius.security.new_api_shop_regex%/.*", role: IS_AUTHENTICATED_ANONYMOUSLY }
+    +   # see https://web.dev/change-password-url/
+    +   sylius_shop_request_password_reset_token_redirect:
+    +       path: /.well-known/change-password
+    +       methods: [GET]
+    +       controller: Symfony\Bundle\FrameworkBundle\Controller\RedirectController::redirectAction
+    +       defaults:
+    +           route: sylius_shop_request_password_reset_token
+    +           permanent: false
```

Comment:
I wonder if this route shouldn't be placed in SyliusShopBundle instead of config/routes/sylius_shop.yaml, but there will be probably a problem with prefixes